daml/Marketplace/Clearing/Service.daml

```diff
@@ -23,27 +20,37 @@ data CalculationResult a b
   = CalculationSuccess { successCid : ContractId a, deposits: Optional DepositWithRemaining}
   | CalculationFailure { failureCid : (ContractId b)}
 
+data DepositAction
```

Comment:
Is this used anywhere?

Reply:
Well spotted, I used it in a previous incarnation but forgot to remove it. 

It's removed in the latest commit.